GitUI/CommandsDialogs/BrowseDialog/DashboardControl/Dashboard.cs

```diff
@@ -161,6 +162,8 @@ void AddLinks(Panel panel, Func<Panel, Control> addLinks, Action<Panel, Control>
 
                 Control CreateLink(Control container, string text, Image icon, EventHandler handler)
                 {
+                    Assumes.NotNull(_selectedTheme);
```

Comment:
I am submitting a PR especially for this, candidate for 3.5

Reply:
Replicated in 0240f7e97f634c054a39998552e14ae8f6d5c7fc.